src/terminfo/parm.rs

```diff
@@ -414,59 +461,64 @@ impl FormatOp {
             FormatOctal => 'o',
             FormatHex => 'x',
             FormatHEX => 'X',
-            FormatString => 's'
+            FormatString => 's',
         }
     }
 }
 
-fn format(val: Param, op: FormatOp, flags: Flags) -> Result<Vec<u8> ,String> {
+fn format(val: Param, op: FormatOp, flags: Flags) -> Result<Vec<u8>, String> {
     let mut s = match val {
-        Number(d) => match op {
-            FormatDigit => {
-                if flags.sign {
-                    format!("{:+01$}", d, flags.precision)
-                } else if d < 0 {
-                    // C doesn't take sign into account in precision calculation.
-                    format!("{:01$}", d, flags.precision + 1)
-                } else if flags.space {
-                    format!(" {:01$}", d, flags.precision)
-                } else {
-                    format!("{:01$}", d, flags.precision)
+        Number(d) => {
+            match op {
+                FormatDigit => {
```

Comment:
Is there any way to tell rustfmt to not put sub-matches on new lines? Putting this match one line up would save an indent.